### PR TITLE
Add tabtweak from MozillaOnline

### PR DESCRIPTION
**Pull request by l-hedgehog**
This is necessary to update the extension in preparation for the `Services.jsm` removal.

[The repo](https://github.com/mozilla-extensions/tabtweak) was just transferred into mozilla-extensions, please also grant [the MozillaOnline team](https://github.com/orgs/mozilla-extensions/teams/mozillaonline/repositories) write access to it.

r?@escapewindow 

Thanks!

### CODE REVIEW

File: taskcluster/ci/config.yml
Comment (jcristau) [inline taskcluster/ci/config.yml:82]: github shows "develop" as the default branch for this repo (with "master" 3 years behind), is this intended?

Comment (l-hedgehog) [reply taskcluster/ci/config.yml:82]: Yes, it's intended. We've always been using `develop` & `master` branches in this way, where the `master` branch is seen as a release branch, and only updated when a new version is ready.

**Review by jcristau — APPROVED** [review]

**Comment by l-hedgehog** [comment]
Now that I have write access to the repo, I've pushed the same commit on `develop` to the `master` branch, and [the TC tasks](https://github.com/mozilla-extensions/tabtweak/runs/8973902487) ran successfully.

Anything else I should do before this could be merged? Thanks!

**Comment by l-hedgehog** [comment]
@jcristau Thank you!

**Pull request merged**
